sim/battle-stream.ts

```diff
@@ -81,6 +81,7 @@ export class BattleStream extends Streams.ObjectReadWriteStream {
 	}
 
 	_writeLine(line: string) {
+		// tslint:disable-next-line:prefer-const
```

Comment:
```
"prefer-const": [true, {"destructuring": "all"}]
```

https://palantir.github.io/tslint/rules/prefer-const/

Reply:
In general, I _hate_ using `tslint:disable` and equivalents, and will prefer basically anything else.